entity_emailer/tests/interface_tests.py

```diff
@@ -413,6 +413,7 @@ class SendUnsentScheduledEmailsTest(TestCase):
     def setUp(self):
         G(Medium, name='email')
 
+    @override_settings(DISABLE_DURABILITY_CHECKING=True)
```

Comment:
I think you can apply this at the class level if you need every method to be overridden

Reply:
oh shit, I only stubbed out the only test in this class that I need to stay undecorated